Koronavilkku/Koronavilkku/View/Home/StatusHeaderView.swift

```diff
@@ -128,14 +130,18 @@ final class StatusHeaderView: UIView {
         fatalError("init(coder:) has not been implemented")
     }
     
-    func adjustSize(by: CGFloat) {
+    func adjustSize(by: CGFloat, topInset: CGFloat) {
```

Comment:
Maybe add comments on how this works and why it does what it does and/or UI tests